main.js

```diff
@@ -85,19 +87,21 @@ function createArticleMarkup(article = {}) {
 function loadNews(source, page) {
 	isLoadingMore = true;
 	fetch(`https://newsapi.org/${apiVersion}/everything?sources=${source}&page=${page}&apiKey=${apiKey}`)
-	  .then((response) =>  response.json())
-	  .then((response) => {
-	  	if (response.status == 'ok') {
-			response.articles.forEach((article) => {
-				let articleContent = createArticleMarkup(article);
-				articlesWrapper.innerHTML += articleContent;
-			});
-	  	}
-	  })
-	  .then(() => {
-  		isLoadingMore = false;
-	  })
-	  .catch(alert);
+		.then((response) =>  response.json())
+		.then((response) => {
+			if (response.status == 'ok') {
+				response.articles.forEach((article) => {
+					let articleContent = createArticleMarkup(article);
+					articlesWrapper.innerHTML += articleContent;
+				});
+			}
+		})
+		.then(() => {
```

Comment:
just interested, how do we have a promise here if in the previous block nothing is returned?

Reply:
it's just for dividing logic